src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md

```diff
@@ -50,11 +50,12 @@ Set-AzVMDiskEncryptionExtension [-ResourceGroupName] <String> [-VMName] <String>
 The **Set-AzVMDiskEncryptionExtension** cmdlet enables encryption on a running infrastructure as a service (IaaS) virtual machine in Azure.
 This cmdlet enables encryption by installing the disk encryption extension on the virtual machine. 
 
-The *VolumeType* parameter is required when encrypting Linux virtual machines, and must be set to a value ("Os", "Data", or "All") supported by the Linux distribution. The *VolumeType* parameter can be omitted when encrypting Windows virtual machines.
```

Comment:
Isn't volume type still required for Linux VMs? @ejarvi @mayank88mahajan

Reply:
Volume type is still required for Linux